web_widget_image_webcam/static/src/js/webcam_widget.js

```diff
@@ -31,9 +31,9 @@ odoo.define('web_widget_image_webcam.webcam_widget', function(require) {
                 image_format: 'jpeg',
                 jpeg_quality: 90,
                 force_flash: false,
-                fps: 45,
-                swfURL: '/web_widget_image_webcam/static/src/js/webcam.swf',
+                fps: 45
```

Comment:
:fire: 
'fps' doesn't seem to be a valid parameter
https://pixlcore.com/read/WebcamJS#configuration

Reply:
I don't know about that, I changed only the 'swfURL' property related stuff. I'll check it.